pages.pt_BR/common/dd.md

```diff
@@ -0,0 +1,32 @@
+# dd
+
+> Converte e copia um arquivo.
+> Mais informações: <https://www.gnu.org/software/coreutils/dd>.
+
+- Cria um USB drive bootável a partir de um arquivo isohybrid (como uma archlinux-xxx.iso) e mostra o progresso:
+
+`dd if={{arquivo.iso}} of=/dev/{{usb_drive}} status=progress`
+
+- Clona um drive para outro drive com 4 MiB block, ignora erros e mostra o progresso:
+
+`dd if=/dev/{{drive_fonte}} of=/dev/{{drive_destino}} bs=4M conv=noerror status=progress`
+
+- Gera um arquivo com 100 bytes randômicos utilizando o kernel random driver:
```

Comment:
Which is most used?
```suggestion
- Gera um arquivo com 100 bytes aleatórios utilizando o kernel random driver:
```

Reply:
both, but "randômico"  is an adaptation of the English

Reply:
I think "aleatórios" is better